src/components/Account/AccountBalances.tsx

```diff
@@ -68,7 +69,8 @@ export default withSubscription({
       return of(null);
     }
     const daoState = dao;
-    const arc = daoState.dao.context;
+    const arc = getArc();
```

Comment:
this change is for all networks ? why ?

Reply:
This is just a minor refactor to bring the code in alignment with how we are supposed to be obtaining the global instance of `Arc`.